### ISSUE TITLE
Can't choose quality

### ISSUE THREAD

**Issue by Bylka** [CiTuX/TD#21]
Usually when you go on a stream (on pc browser) there options: low medium High etc. On these streams i can choose quality in app.
But if quality options look like this 360p 480p 720p 1080p 720p30 whichever quality i choose in app it goes for best.1080p quality stream requires high connection speed, which i have not.

**Comment by dachacha** [comment]
Yeah I hope for the update too, my phone is really old and this app is the only I can use

**Comment by iillyyaa2033** [comment]
Same issue. Nexus 7 2012, android 6, app version 0.5.0

**Comment by N0Lif3** [comment]
Please update TD to use the new Twitch quality options. As said in this thread, TD is defaulting to the Best quality over other choices available.

I am unable to watch high quality 60fps streams on my old tablet, they just won't play for me. I need to be able to use lower quality options. Please help us, update TD, and roll out the update to F-Droid.

Thank you. TD is one of my favorite android apps.

**Comment by N0Lif3** [comment]
So, I decided to take it upon myself to update TD to have bring back quality settings and after much hassle, I've been able to do it!
![screenshot_2017-06-06-06-58-56](https://cloud.githubusercontent.com/assets/25230710/26833021/108b0384-4a86-11e7-9583-8b8f42ac75cb.png)
New quality settings
![screenshot_2017-06-06-06-59-28](https://cloud.githubusercontent.com/assets/25230710/26833031/1b15e224-4a86-11e7-9c1e-df1ce19be317.png)
ugly 160p screenshot showing you I have quality options again (it isn't defaulting to Source)
 
Here's the apk
[TD-master-debug-unaligned.zip](https://github.com/CiTuX/TD/files/1055238/TD-master-debug-unaligned.zip)

**Comment by dachacha** [comment]
works great! Thank you!

**Comment by iillyyaa2033** [comment]
@N0Lif3 thx, it works! Will you do a pull-request?

**Comment by CiTuX** [comment]
@N0Lif3 Thank you 👍 A PR would be perfect!

**Comment by Bylka** [comment]
I still cant use app ( i cant install modified version because i cant set root on my phone

**Comment by IriaSomobu** [comment]
@Bylka just delete your TD and then install modified

**Comment by Bylka** [comment]
OMG it works now thanks a lot !

**Comment by IriaSomobu** [comment]
@Bylka you are welcome.

**Comment by ghost** [comment]
I don't know how to do a pull request. If somebody would else would like to do it, getting TD working again just meant editing two files. Change the String variables from Mobile / Low / Medium / High to 160p30, 360p30, 480p30, and 720p30 respectively.  It was as easy as that, I changed nothing else to get it working.
